test/force-auto.py.in

```diff
@@ -130,6 +130,12 @@ class Test(abc.ABC):
       # architecture excludes
       arch_excludes = "\n".join("arch_exclude %s" % i
                                 for i in self.arch_excludes)
+      if (str(self) == "alpine:latest, needed, fakeroot, no preprep"):
+         bucache_reset = f"""\
+# Test assumes alpine:latest is absent from bucache. Make it so.
+ch-image build-cace --reset\n"""
```

Comment:
So this has a typo and never actually ran because it's full scope. ;) Also it seems kind of heavy-handed to throw away the entire build cache for one test. If you really do need to run this test w/o cache, you could add `--rebuild`. However I think you may want to just change the `$output` comparisons?